### ISSUE TITLE
TFP distributions - unclear which kwargs the args correspond to

### ISSUE THREAD

**Issue by georgematheos** [genjax-community/genjax#1368]
I always get confused with when calling `genjax.bernoulli`, it takes a probs or logits argument.  The reason why is that `tfp.Bernoulli` can accept either as a kwarg and I don't know which is the "first arg" for when I call `genjax.bernoulli.sample`.  I also don't know how to pass the `Bernoulli` kwargs through into the `genjax.bernoulli.sample` call.

**Comment by georgematheos** [comment]
Hah, actually, I really don't know what genjax.bernoulli does.  Neither of calling it with arg 1.0, or arg 0.0, seem to make it return 1 with 100% probability!

**Comment by femtomc** [comment]
It's a logit to `genjax.bernoulli` -- there's a `genjax.flip` which accepts _probabilities_

**Comment by femtomc** [comment]
See further: 
https://genjax.gen.dev/library/generative_functions.html?h=bernoulli#genjax.generative_functions.distributions.bernoulli
and
https://genjax.gen.dev/library/generative_functions.html?h=bernoulli#genjax.generative_functions.distributions.flip

which both use the underlying `tensorflow_probability.tfd.Bernoulli`

**Comment by sritchie** [comment]
@georgematheos when we get back from break we can tackle this along with the other issue you filed...

**Comment by sritchie** [comment]
@georgematheos I'll tackle this one with a docs clarification, and then we'll solve the related, bigger issue — that kwargs don't work reliably — over at https://linear.app/chisym/issue/GEN-889/keyword-args-arent-tested-dont-make-their-way-down-to-primitives.

**Comment by sritchie** [comment]
Closed via https://github.com/ChiSym/genjax/pull/1526
